barretenberg/cpp/src/barretenberg/stdlib_circuit_builders/mega_flavor.hpp

```diff
@@ -39,24 +39,23 @@ class MegaFlavor {
     static constexpr size_t NUM_WIRES = CircuitBuilder::NUM_WIRES;
     // The number of multivariate polynomials on which a sumcheck prover sumcheck operates (including shifts). We often
     // need containers of this size to hold related data, so we choose a name more agnostic than `NUM_POLYNOMIALS`.
-    // Note: this number does not include the individual sorted list polynomials.
-    static constexpr size_t NUM_ALL_ENTITIES = 58;
+    static constexpr size_t NUM_ALL_ENTITIES = 57;
```

Comment:
Explanation of the changes in these numbers: We gained three polys: `lookup_inverses`, `lookup_read_counts`, `lookup_read_tags`. None of these need to be shifted. We lost sorted_accum and z_lookup, BOTH of which need to be shifted. So the ALL entities gets bumped by 3 - 4 = -1 and the witness entities gets bumped by 3 - 2 = 1.